zipline/assets/asset_writer.py

```diff
@@ -803,7 +803,7 @@ def _write_df_to_table(self, tbl, df, txn, chunk_size):
 
         df.to_sql(
             tbl.name,
-            txn.connection,
+            txn,
```

Comment:
txn.connection works with sqlite but not with postgres. if we're using postgres it will return a ConnectionFairy-object which pandas later on can't handle when writing to sql. Both works for sqlite